src/transforms/filter.jl

```diff
@@ -69,22 +65,43 @@ DropMissing(cols::T...) where {T<:ColSelector} =
 
 isrevertible(::Type{<:DropMissing}) = true
 
-_ftrans(::DropMissing{Colon}, table) =
+# ftrans
+_ftrans(::DropMissing{Colon}, cols) =
   Filter(row -> all(!ismissing, row))
 
-function _ftrans(transform::DropMissing, table)
-  allcols = Tables.columnnames(table)
-  cols = _filter(transform.colspec, allcols)
+_ftrans(::DropMissing, cols) =
   Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
-end
+
+# nonmissing 
+_nonmissing(::Type{T}, x) where {T} = x
+_nonmissing(::Type{Union{Missing,T}}, x) where {T} = collect(T, x)
+_nonmissing(x) = _nonmissing(eltype(x), x)
 
 function apply(transform::DropMissing, table)
-  ftrans = _ftrans(transform, table)
+  colnames = Tables.columnnames(table)
+  selnames = _filter(transform.colspec, colnames)
+  ftrans = _ftrans(transform, selnames)
   newtable, fcache = apply(ftrans, table)
-  newtable, (ftrans, fcache)
+
+  # post-processing
+  coltable = Tables.columntable(newtable)
+  pcolumns = [nm ∈ selnames ? _nonmissing(x) : x for (nm, x) in pairs(coltable)]
```

Comment:
```suggestion
  pcolumns = [n ∈ selnames ? _nonmissing(x) : x for (n, x) in pairs(coltable)]
```

Reply:
Made in [1514725](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/151472520bcb0c3d5222738bfa33dce003172cf7)